src/store/undoRedo/index.ts

```diff
@@ -115,15 +124,29 @@ export const actions: ActionTree<IUndoRedoState, IRootState> = {
       flows: rootState.flow,
     })
 
+    let isLastFlowStateFromPersistence = !getters.hasCurrentSnapshot
+    const currentSavedAt = getters.currentSnapshot?.modules.flows?.savedAt
+    const nextSavedAt = nextSnapshot?.modules.flows?.savedAt
```

Comment:
would this be better naming?
`isLastFlowStateFromPersistence` -> `isCurrentFlowStateFromPersistence` ('last' is associated with the snapshot stack to me again, but we are talking about state)
`currentSavedAt` -> `oldSavedAt`
`nextSavedAt` -> `newSavedAt`